akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/AbstractBehavior.scala

```diff
@@ -70,12 +70,13 @@ abstract class AbstractBehavior[T](protected val context: ActorContext[T]) exten
   @throws(classOf[Exception])
   def onSignal: PartialFunction[Signal, Behavior[T]] = PartialFunction.empty
 
-  private def checkRightContext(ctx: TypedActorContext[T]): Unit =
+  private def checkRightContext(ctx: TypedActorContext[T]): Unit = {
     if (ctx.asJava ne context)
       throw new IllegalStateException(
         s"Actor [${ctx.asJava.getSelf}] of AbstractBehavior class " +
         s"[${getClass.getName}] was created with wrong ActorContext [${context.asJava.getSelf}]. " +
         "Wrap in Behaviors.setup and pass the context to the constructor of AbstractBehavior.")
```

Comment:
Will this two `s` be optimized as one?

Reply:
in plain Java they would, I don't know about the string interpolators, but anyway not important because this is not supposed to happen often (would be bug in application)